src/scancode/cli.py

```diff
@@ -229,14 +229,18 @@ def validate_formats(ctx, param, value):
 @click.option('--timeout', is_flag=False, default=DEFAULT_TIMEOUT, type=int, show_default=True, help='Stop scanning a file if scanning takes longer than a timeout in seconds.')
 @click.option('--max-memory', is_flag=False, default=DEFAULT_MAX_MEMORY, type=int, show_default=True, help='Stop scanning a file if scanning requires more than a maximum amount of memory in megabytes.')
 
+#--ignore accounts for ignoring user-specified files during the scanning process.
+@click.option('--ignore', show_default=True, default='', metavar='<pattern>', multiple=True, help='Ignore files mentioned with --ignore while scanning.')
```

Comment:
`multiple=True` is the way to go. But you need to find a better help text. @sschuberth any input?